modules/server/README.md

```diff
@@ -18,3 +18,9 @@ Build (required before publish):
 ```
 npm run prepare
 ```
+
+## Federated Search
+
+### Config
+
+Pass `true` to the environment variable `ENABLE_NETWORK_AGGREGATION` to enabled federated search.
```

Comment:
quick typo. however, I'm thinking how this will need to be extended into profiles, rather than on/off (boolean)

```suggestion
Pass `true` to the environment variable `ENABLE_NETWORK_AGGREGATION` to enable federated search.
```

Reply:
Not 100% clear on what profiles entails. 
For this PR I think its ok to just have something there to say "this functionality is based on env"
let me know if I can create a ticket re profiles